example/lib/main.dart

```diff
@@ -35,12 +35,12 @@ class _QRViewExampleState extends State<QRViewExample> {
               key: qrKey,
               onQRViewCreated: _onQRViewCreated,
               overlay: QrScannerOverlayShape(
-                borderColor: Colors.red,
-                borderRadius: 10,
-                borderLength: 30,
-                borderWidth: 10,
-                cutOutSize: 300,
-              ),
+                  borderColor: Colors.red,
+                  borderRadius: 10,
+                  borderLength: 30,
+                  borderWidth: 10,
+                  cutOutSize: 300,
+                  cutOutBottomOffset: 24),
```

Comment:
```suggestion
                  cutOutBottomOffset: 24,),
```
and `dartfmt` please.